src/mr.roboto/src/mr/roboto/subscriber.py

```diff
@@ -256,6 +257,9 @@ def run(self):
                 "please allow a week for your agreement to be processed.\n"
                 "Once it is processed, you will receive an email invitation "
                 "to join the `plone` GitHub organization as a Contributor."
+                "\n\n"
+                "If after a week, you have not received an invitation, then "
```

Comment:
Sorry, I was in a rush, and my grammar was weak.

```suggestion
                "If after a week you have not received an invitation, then "
```